polkadb/interface.go

```diff
@@ -0,0 +1,15 @@
+package polkadb
+
+// PutIt wraps the database write operation supported by regular database.
```

Comment:
Why?

Reply:
My main reasoning for this is when I implement the `Batch` interface I can reference the `PutIt` interface rather than writing out the entire Put method declaration twice as the Put method is the only shared method between the `Database` interface and what will be the `Batch` interface

Reply:
@ansermino

Reply:
I think that makes sense, it would just be nice to some air of this reflected in the comment. The snippet `supported by regular database` seems to be fluff